src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -169,7 +128,7 @@ export const WorkspaceForm = ({
       ? WorkspaceFormTabs.UsersAndPermissions
       : WorkspaceFormTabs.NotSelected
   );
-  const [numberOfErrors, setNumberOfErrors] = useState(0);
+  const [errorsCount, setErrorsCount] = useState(0);
```

Comment:
Could we calculate errorsCount based formErrors, we can wrap with useMemo. Then we don't need a separate state.

Reply:
Since errorsCount is based on formErrors, shall I declare useState for formErrors?

Reply:
Can't get the point, just want to refactor this errorsCount state to
```tsx
const errorsCount = useMemo(()=> getErrorsCount(formErrors), [formErrors])
```